src/npmPackageData/retrievePackageDataByName.mjs

```diff
@@ -0,0 +1,16 @@
+import getRoadieStore from './getRoadieStore.mjs';
+
+const retrievePackageDataByName = async ({
+  packageName,
+}) => {
+  if (typeof packageName === 'undefined' || packageName === '') {
+    throw new Error(`
+      Argument packageName '${packageName}' must not be undefined or an empty string.
```

Comment:
Netligy blobs seems to return the whole datastore contents if you use `#get` with `undefined` in the first argument. 🤷